core/src/main/java/com/rudderstack/core/Configuration.kt

```diff
@@ -69,7 +69,7 @@ interface Configuration {
             sdkVerifyRetryStrategy: RetryStrategy = RetryStrategy.exponential(),
             dataPlaneUrl: String? = null, //defaults to https://hosted.rudderlabs.com
             controlPlaneUrl: String? = null, //defaults to https://api.rudderlabs.com/
-            logger: Logger = KotlinLogger,
+            rudderLogger: RudderLogger = KotlinLogger(RudderLogger.LogLevel.NONE),
```

Comment:
Let's remove the initial value that is set through the constructor.

Reply:
I missed this somehow!